samples/WebRequestTracerSamples.cs

```diff
@@ -27,7 +27,7 @@ static class WebRequestTracerSamples
     {
         public static void OutgoingWebRequest()
         {
-            MyCustomHttpRequest request = new MyCustomHttpRequest("https://www.example.com:8080/api/user?group=42&location=Linz", "GET");
+            MyCustomHttpRequest request = new MyCustomHttpRequest("https://www.example.com:8080/api/auth/user?group=42&location=Linz", "GET");
```

Comment:
Should we use `var` for "fully-redundant newing"? Not sure what we settled on ...